src/components/frame/v5/pages/VerifiedPage/partials/VerifiedTable/VerifiedTable.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { FC, useState } from 'react';
+import { useIntl } from 'react-intl';
+
+import {
+  getPaginationRowModel,
+  getSortedRowModel,
+  SortingState,
+} from '@tanstack/react-table';
+import Filter from '~v5/common/Filter';
+import Button from '~v5/shared/Button';
+import { TableProps } from './types';
+import EmptyContent from '~v5/common/EmptyContent';
+import { useSearchContext } from '~context/SearchContext';
+import { useVerifiedTableColumns } from './hooks';
+import Table from '~v5/common/Table';
+import { useMobile } from '~hooks';
+import TableHeader from '~v5/common/TableHeader';
+import { formatText } from '~utils/intl';
+
+const displayName = 'v5.pages.VerifiedPage.partials.VerifiedTable';
+
+const VerifiedTable: FC<TableProps> = ({ list, name }) => {
+  // @TODO: Add action for adding new member, removing user from whitelist, handle pagination
+  const { formatMessage } = useIntl();
+  const isMobile = useMobile();
+  const { searchValue } = useSearchContext();
+
+  const onAddClick = () => {};
+  const columns = useVerifiedTableColumns(name);
+
+  const listLength = list.length;
+  const [sorting, setSorting] = useState<SortingState>([]);
+  const [rowSelection, setRowSelection] = useState({});
+
+  return (
+    <>
+      <TableHeader
```

Comment:
I'm wondering if it would be better to add this as a part of the table component? What do you think guys?